frontend/packages/shipwright-plugin/src/actions/useBuildActions.ts

```diff
@@ -36,8 +36,30 @@ const useBuildActions = (build: Build) => {
         namespace: build.metadata?.namespace,
       },
     };
-
-    return [start, ...getCommonResourceActions(kindObj, build)];
+    const startLastRun: Action = {
+      id: 'shipwright-build-start-last-run',
+      label: t('shipwright-plugin~Start last run'),
+      disabled: !build.latestBuild,
+      cta: () => {
+        rerunBuildRun(build.latestBuild)
+          .then((newBuildRun) => {
+            history.push(resourceObjPath(newBuildRun, referenceFor(newBuildRun)));
+          })
+          .catch((err) => {
+            const error = err.message;
+            errorModal({ error });
+          });
+      },
+      accessReview: {
+        verb: 'create',
+        group: BuildRunModel.apiGroup,
+        resource: BuildRunModel.plural,
+        namespace: build.metadata?.namespace,
+      },
+    };
+    return build.latestBuild
+      ? [start, startLastRun, ...getCommonResourceActions(kindObj, build)]
+      : [start, ...getCommonResourceActions(kindObj, build)];
```

Comment:
I personally would create an `actions: Action[]` array and then would add elements if they are available or not and return that because that's more extendable. Its also not needed to create an action if there is no `build.latestBuild`. But I'm also fine with this if you want keep it.

Like

```tsx
  const actions: Action[] = [];
  actions.push({
      id: 'shipwright-build-start',
      label: t('shipwright-plugin~Start'),
      // ...
  });
  if (build.latestBuild) {
    actions.push({
      id: 'shipwright-build-start-last-run',
      label: t('shipwright-plugin~Start last run'),
      // ...
    });
  }
  actions.push(...getCommonResourceActions(kindObj, build))
  return actions;
```